fhir-examples/src/main/java/com/ibm/fhir/examples/Index.java

```diff
@@ -90,15 +101,28 @@
      */
     PROFILES_PDEX_PLAN_NET_JSON("/profiles-pdex-plan-net-json.txt"),
 
+    /**
+     * US Core Examples in XML and JSON
+     */
+    PROFILES_US_CORE_JSON("/profiles-us-core-json.txt"),
```

Comment:
pretty minor, but this will be taken as javadoc for only PROFILES_US_CORE_JSON.  i'd suggest we either change it to a block comment (removing a single `*`) or just have one for each.
```suggestion
    /*
     * US Core Examples in XML and JSON
     */
    PROFILES_US_CORE_JSON("/profiles-us-core-json.txt"),
```

Reply:
sometimes for comments like these I prefer the old-fashioned `//` to make it clear its not javadoc:
```suggestion
    ///////////////////////////////////
    // US Core Examples in XML and JSON
    ///////////////////////////////////
    PROFILES_US_CORE_JSON("/profiles-us-core-json.txt"),
```

Reply:
Fixed locally and bundled into one PR